### DISCUSSION TITLE
Error when i run my app

### DISCUSSION THREAD

**Discussion by Melisayg12** [Homebrew/discussions#3590] [Everyday usage]
Hi, I have this error when I try to run my app.
I put two commands:
yarn run mongo (successfully executed)
yarn run maps (maps is my app)
<img width="1296" alt="187438893-b1ce1fe8-b461-4e57-89ff-832d33b330a9" src="https://user-images.githubusercontent.com/99819169/187612421-702374f7-a17d-44eb-802e-eecab9f9ddc3.png">

myanes@MacBook-Pro-de-Melisa-2 ipsilum % yarn run maps
yarn run v1.22.19
$ cd maps && env MONGO_URL=mongodb://Localhost: 27017/ipsilum NODE_TLS_REJECT_UNAUTHORIZED=0 meteor --settings .deploy/settings-dev.json --port 3004
This project uses Meteor 1.8.3, which isn't available on this platform. To work with this app on all supported platforms, use meteor update -release METEOR@2.7.3 to pin this app to the newest compatible release.
error Command failed with exit code 1
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
• myanes@MacBook-Pro-de-Melisa-2 ipsilum % meteor --version
Meteor 2.7.3

**Comment by carlocab** [comment]
You've already posted this at #3586. Please don't make duplicate posts over a short period of time.
